README.zh-CN.md

```
@@ -0,0 +1,166 @@
+<div align="left">
+
+  <a href="https://aimeos.org/">
+    <img src="./apps/deploy-web/public/android-chrome-192x192.png" alt="Akash logo" title="Akash Console" align="left" height="40" />
+  </a>
+
+  # Akash Console
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)